file-availability.md

```diff
@@ -1,61 +1,62 @@
+data, code, packages, software, etc. from the submit server where the job 
+is submitted to the execute node where the job will run. This requirement 
+applies to any and all files that are needed to successfully execute and 
+complete your job.
+
+Any output that gets generated by your jobs is specifically written to 
+the execute node on which the job ran. In order to get access to 
+your output files, a copy of the output must be transferred back 
+to an user accessible location like the submit server.
+
+**The mechanism that is used for file transfers will depend on the size 
+of the individual input and output files of your jobs.** This guide 
+specifically describes input and output file transfer for input files 
+<100MB in size (and <500MB of total input file transfer) and output 
+files <4GB in size using the standard solution built into HTCondor 
+job scheduling. More information about file transfer on a system 
+without a shared filesystem is available in the 
+[HTCondor manual](https://htcondor.readthedocs.io/en/latest/users-manual/file-transfer.html).
 
 <a name="Appli"></a>
 
-**1. Applicability**
-----------------
+# Applicability
 
 * Intended use:  
 Good for delivering any type of data to jobs, but with file-size
 limitations (see below). Remember that you can/should split up a large
 input file into many smaller files for cases where each job only needs a
-portion of the data. By default, the submit file \"executable\",
-\"output\", \"error\", and \"log\" files are ALWAYS transferred.
+portion of the data. By default, the submit file `executable`,
+`output`, `error`, and `log` files are **ALWAYS** transferred.
 
 * Advantages:  
-HTCondor\'s file transfer is robust and is available on ANY of CHTC\'s
-accessible HTC resources (including the UW Grid of campus pools, and the
-Open Science Grid).
-
-* Input File Limitations:  
-HTCondor\'s file transfer can cause issues for submit server performance
-when too many jobs are transferring too much data at the same time.
-Therefore, HTCondor file transfer is only good for input files up to
-\~20 MB per file IF the number of concurrently-queued jobs will be 100
-or greater. Even when individual files are small, there are issues when
-the total amount of input data per-job approaches 500 MB. For cases
-beyond these limitations, one of our other CHTC file delivery methods
-should be used. Remember that creating a \*.tar.gz file of directories
-and files can give your input and output data a useful amount of
-compression.
-
-* Output File Limitations:  
-Because jobs are less likely to be completing at the same time, total
-job output size of up to 1 GB will not cause submit server performance
-issues, but it\'s always advantageous to create a \*.tar.gz file of all
-desired output before job completion (and to also delete the un-tar\'d
-files so they are not also transferred back).
+HTCondor file transfer is robust and is available on ANY of CHTC\'s
```

Comment:
do you need the `\`?

Reply:
idk, most of the apostrophe's were escaped in the previous version